install/0000_90_machine-api-operator_05_prometheusrules-ironic-prometheus-exporter.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: metal3-baremetalhost-controller
+  namespace: openshift-machine-api
+spec:
+  groups:
+  - name: metal3-baremetalhost-controller
+    rules:
+    - alert: HighCPUTemperature
+      annotations:
+        summary: "The baremetal node {{ $labels.node_name }} CPU {{ $labels.entity_id }} is too high"
+        description: "The baremetal node {{ $labels.node_name }} CPU {{ $labels.entity_id }} was too high in the past 5 minutes. Last measurement {{ $value }}"
```

Comment:
wouldn't this trigger for non baremetal envs?

Reply:
Should porbably rename `syncBaremetalControllers` to something more generic e.g `syncBaremetal`
Then `syncBaremetal`  calls `syncBaremetalControllers`  and `syncBaremetalMonitoring`

Reply:
@enxebre I don't think it would trigger in non baremetal envs, check @simonpasquier comment. 
The sync part you are talking would be if we make the yamls go code right?